src/background.js

```diff
@@ -726,6 +654,20 @@ function reportCliqzOffer(message) {
  * @return {boolean}            denotes async (true) or sync (false)
  */
 function onMessageHandler(request, sender, callback) {
+	if (request.module === 'offers-banner' && request.action === 'send') {
```

Comment:
@christophertino this does not seem great here, but not sure where else fits better. any suggestion?

Reply:
just use ```import { sendRewardMessage } from '../utils/msg';```